common/Mesh-Network-Joining-Process-Comparison.md

```diff
@@ -0,0 +1,540 @@
+  - [2.2. 802.15.4 Network topology](#22-802154-network-topology)
+    - [2.2.1. Device type in IEEE 802.15.4 network](#221-device-type-in-ieee-802154-network)
+    - [2.2.2. Star and Peer-to-Peer topology](#222-star-and-peer-to-peer-topology)
+  - [2.3. Mesh Network maintenance](#23-mesh-network-maintenance)
+    - [2.3.1. Device type in Zigbee network](#231-device-type-in-zigbee-network)
+    - [2.3.2. Establish a New Network](#232-establish-a-new-network)
+    - [2.3.3. Permit Device to Join a Network](#233-permit-device-to-join-a-network)
+    - [2.3.4. Network Discovery](#234-network-discovery)
+    - [2.3.5. Joining a Network](#235-joining-a-network)
+      - [2.3.5.1. Joining a secured network](#2351-joining-a-secured-network)
+      - [2.3.5.2. Authorization](#2352-authorization)
+- [3. Bluetooth Mesh](#3-bluetooth-mesh)
+  - [3.1. Architecture Introduction](#31-architecture-introduction)
+  - [3.2. Network topology](#32-network-topology)
+  - [3.3. Node type in Bluetooth Mesh network](#33-node-type-in-bluetooth-mesh-network)
+  - [3.4. Provisioning behavior](#34-provisioning-behavior)
+    - [3.4.1. Beaconing](#341-beaconing)
+      - [3.4.1.1. PB-ADV](#3411-pb-adv)
+      - [3.4.1.2. PB-GATT](#3412-pb-gatt)
+    - [3.4.2. Invitation](#342-invitation)
+    - [3.4.3. Exchanging public keys](#343-exchanging-public-keys)
+    - [3.4.4. Authentication](#344-authentication)
+    - [3.4.5. Distribution of the Provisioning Data](#345-distribution-of-the-provisioning-data)
+- [4. Conclusion](#4-conclusion)
+  - [4.1. Network joining process in Zigbee](#41-network-joining-process-in-zigbee)
+  - [4.2. Provisioning process in Bluetooth Mesh](#42-provisioning-process-in-bluetooth-mesh)
+</details>
+
+***
+
+# 1. Background
+Silicon Labs is developing products designed to meet the demands of customers as we move to an ever-connected world of devices in the home, what is often referred to as the IoT (Internet of Things). And we offers the Flex, EmberZNet, BLE, Bluetoooth Mesh and Z-Wave stack for development on the Wireless Gecko family to meet different requirement from customer.   
+
+This document describes the different process about how a device joining the mesh network in Zigbee and Bluetooth Mesh protocol.   
+
+***
+
+# 2. Zigbee
+## 2.1. Architecture Introduction
+As you already know that the Zigbee networks are based on the IEEE 802.15.4 MAC and physical layer. The figure below illustrates the architecture of Zigbee.   
+
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-zigbee-architecture.png">  
+</div>  
+
+The 802.15.4 MAC layer is used for basic message handling and congestion control. This MAC layer includes mechanisms for forming and joining a network, a CSMA mechanism for devices to listen for a clear channel, as well as a link layer to handle retries and acknowledgment of messages for reliable communications between adjacent devices.   
```

Comment:
1. as well as a link layer
[Mark] No link layer shown in the image
2. CSMA
[Mark] Carrier Sense Multiple Access (CSMA)